static/bidder-info/orbidder.yaml

```diff
@@ -6,3 +6,11 @@ capabilities:
   app:
     mediaTypes:
       - banner
+  site:
+    mediaTypes:
+      - banner
+      - native
+userSync:
+  redirect:
+    url: "https://orbidder.otto.de/pbs-usersync?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}"
+    userMacro: "[ODN_ID]"
```

Comment:
<img width="759" alt="Screenshot 2023-07-14 at 11 06 54 AM" src="https://github.com/prebid/prebid-server/assets/8147531/45719936-4863-4efa-81eb-0b07261de891">
UID is not being set and returning 404 status code. Please validate the user sync url.

Here is the URL that I am trying to use - 
```
https://orbidder.otto.de/pbs-usersync?gdpr=&consent=&us_privacy=&redirect=https%3A%2F%2Fib.adnxs.com%2F%2Fsetuid%3Fbidder%3Dorbidder%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%5BODN_ID%5D
```

Reply:
Hello, one question, is the redirect url correct?

When i change:
`https://ib.adnxs.com//setuid?bidder=orbidder&gdpr=&gdpr_consent=&gpp=&gpp_sid=&f=i&uid=`
to:
`https://ib.adnxs.com/setuid?bidder=orbidder&gdpr=&gdpr_consent=&gpp=&gpp_sid=&f=i&uid=`
it looks fine to me.

When adding a grpr=1 and a consent string i could see that the redirect contains my user id.

Have a nice day

Reply:
![image](https://github.com/prebid/prebid-server/assets/22349602/a781ffd7-512f-4a17-9952-6621cea0b6df)

Reply:
The id is still not yet set. I have tried with the following URL as well - 
```
https://orbidder.otto.de/pbs-usersync?gdpr=&consent=&us_privacy=&redirect=https%3A%2F%2Fib.adnxs.com%2Fsetuid%3Fbidder%3Dorbidder%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%5BODN_ID%5D
```
<img width="1792" alt="Screenshot 2023-07-14 at 1 55 24 PM" src="https://github.com/prebid/prebid-server/assets/8147531/2327d7f6-043f-495a-84cc-e114af631a1d">

Reply:
Hello, our backend will read a cookie that will be set when visiting "https://www.otto.de/" and accepting the cookie overlay. If somebody never visit "https://www.otto.de/" once, we will redirect with an empty userid, because we do not know this user.

In this case, that we do not know the user, should we do something different then redirecting with a empty uid?
Maybe answer the request with "204" instead of a redirect?

Reply:
Even if I visit "https://www.otto.de/" first and then call the user sync endpoint, It is still not working for me.
<img width="1784" alt="Screenshot 2023-07-19 at 4 18 48 PM" src="https://github.com/prebid/prebid-server/assets/8147531/21949c3a-2028-442f-b934-d75d1649e6a4">

and https://www.otto.de/ was open in the other browser tab before hitting the above user sync URL.

Reply:
This is the recommended action. If you have never seen the user before, consider this their first visit and generate a user id. The user id can be returned to the browser as a cookie on the response and embedded in the redirect url to inform Prebid Server.

We will be updating the Prebid documentation to make this expectation clear.

Reply:
@SyntaxNode @gargcreation1992, as commented below, we delete the sync-endpoint for now. https://github.com/prebid/prebid-server/pull/2924/commits/83654303df98c474e6742f79aad74a8d999dcb27

Because at the moment we can not fulfill the requirements to set the user-id for users that we do not know and we need the native changes merged and migrated to the java prebid server version.

@gargcreation1992, to you accept the cookie terms and conditions at the bottom of the site?
Because "otto" is only allowed to set the needed cookie after accepting the cookie conditions.